lib/nexpose/common.rb

```diff
@@ -25,20 +25,25 @@ module Scope
   # non-members.
   class Email
     # Send as file attachment or zipped file to individuals who are not members
-    # of the report access list. One of: file|zip
+    # of the report access list. One of: [String] 'file' | 'zip'
```

Comment:
Might be clearer as:
```
[String] the attachment format, `file` or `zip`.
```
Possibly on a new line rather than being the end of the 2nd line.